src/CargoOrder.java

```diff
@@ -1,4 +1,24 @@
 package traingame;
 
+import java.util.Random;
+
 public record CargoOrder(Product product, City destination, int payment) {
+    //NOTE: For now, keeping it simple.  Just random.  Might want to re-balance payout later.
+    //For example, to care about the distance between pickup and delivery cities.
+	public static CargoOrder getRandom(City[] cities) {
+        //Chose a random product, city and payout.
+        Random random = new Random();
+        Product chosenProduct = Product.valueOf(Product.getRandom().name());
```

Comment:
There's a shorter way to do this. Think about the return type of each function called here.

Reply:
Right,  I can use Product.getRandom() instead as that should already return a Product.